### PR TITLE
[CREATED]: Created basic structure for login page

### PR DESCRIPTION
**Pull request by VishruthVS**
->Vishruth VS
->Created the basic working structure for login page

### CODE REVIEW

**Review by avinasdube — APPROVED** [review]
No issues. Good to go for merge.

**Comment by avinasdube** [comment]
Hi @VishruthVS ,

Thank you for your pull request! I appreciate your contribution to the project.

Your changes do not conflict with existing ones. But I'd like to suggest you following things : 

1. Next time when you make PR, do include screenshots (if you have) of working changes in the PR.
2. Create separate styling file for each separate page (if you work on one).
3. Include proper comments on necessary lines. 

Best regards,

Avinash Dubey
Repository Maintainer/Manager

File: sa-client/src/App.css [hunk 1]
```diff
@@ -1,15 +1,44 @@
-body{
-  margin: 0;
-}
+/* LoginForm.css */
```
Comment (dhanuj00123) [inline sa-client/src/App.css:1]: this should be moved to LoginForm.scss

Comment (VishruthVS) [reply sa-client/src/App.css:1]: alright

**Pull request merged**